vendor/teleopti/calendar-data-updater.directive.js

```diff
@@ -0,0 +1,159 @@
+(function() {
```

Comment:
Yah I agree. I have also seen Rob and Zhiping trying to add another directive in their pull request. I think we need a standard place to put all those self-made directives. Could you create one?